Say/ViewController.swift

```diff
@@ -131,35 +131,24 @@ class ViewController: NSViewController {
     
     @IBAction func selectText(_ sender: NSTextField) {
         if let url = URL(string: sender.stringValue) {
+            let instapaper = "https://www.instapaper.com/text?u="
+            let onlyText = URL(string:"\(instapaper)\(url)")
```

Comment:
The name `onlyText` looks like `String` object. I recommend to use `textURL` or `instapaperURL`.